automation/.gitignore

```
@@ -1,6 +1,12 @@
+# Add output directory
+output/
+
 # Add test environment
 test_env/
 
+# DS Store for Mac
+.DS_Store
```

.DS_Store files can be created in sub-folders too.

```suggestion
*.DS_Store
```